docs/conf.py

```diff
@@ -63,10 +63,10 @@
 html_last_updated_fmt = ''  # Suppress 'Last updated on:' timestamp
 html_show_sphinx = False
 html_favicon = os.path.join("_static", "favicon.ico")
-html_sidebars = {"**": ["sidebar.html"]}
+html_sidebars = {"**": []}
 html_theme_options = {
-    "sidebar_logo_width": 200,
-    "nengo_logo": "general-full-light.svg",
+    "sidebar_logo_width": 20,
+    "nengo_logo": "general-square-light.svg",
```

Comment:
Is this used anywhere? I didn't see it when I rendered the docs (since we don't have the sidebar anymore, I assume). I'm not worried (docs look fine), just curious.